### ISSUE TITLE
Choose Search Engine Option

### ISSUE THREAD

**Issue by Exotic-Code** [brave/browser-laptop#13651]
### Description

Brave should have a option to choose a search engine not have a default one (currently Google)
So that you "boot up" Brave for the first time you can choose which search engine you use as the default.

I made a image to illustrate what I mean:
[![Selection_109.png](https://s14.postimg.org/4tugbs83l/Selection_109.png)](https://postimg.org/image/sktttw8al/)

**Comment by Exotic-Code** [comment]
[
  "Label1",
  "Label2"
]

**Comment by jonathansampson** [comment]
Thank you for the suggestion, @Exotic-Code! Something to consider for the about:welcome page :)

**Comment by hammok** [comment]
@Exotic-Code - one potential work-around is if you default search to duckduckgo you can use their !Bang feature to easily access numerous search engines with an easy prefix:

https://duckduckgo.com/bang

You get the added benefit of defaulting to a privacy-oriented browser.

**Comment by Exotic-Code** [comment]
@hammok Oh, I think you misunderstood.
I use DuckDuckGo and its !bangs as a daily driver.

Brave say they can't change the default away from Google because of some users dont know that it exist alternatives. So they have kept it as Google for now.
But i feel thats unfear for the other search engines so I want an option to appear when the user for the first time opens Brave to choose the default search engine not just set Google as a default.

**Comment by hammok** [comment]
@Exotic-Code Yep, misunderstood.  :-) . Or the default should just be duck duck go!  :-)

**Comment by Liunkae** [comment]
Related: #9748 (Make DuckDuckGo the default search engine)

**Comment by Liunkae** [comment]
Quote from @bradleyrichter https://github.com/brave/browser-laptop/issues/9748#issuecomment-347392111:
> We will be doing a v2 redesign on the welcome page and will absolutely add a search engine default section. This is a great place to present it to the new user.
